examples/gltf/app.js

```diff
@@ -60,14 +60,15 @@ const INFO_HTML = `
 
 export const animationLoopOptions = {
   models: [],
+  eye: [0, 0, 1],
   gl: null,
   loadedModelUrl: null,
 
   loadGLTF: url => {
     window.fetch(url).then(res => res.arrayBuffer()).then(data => {
 
       const gltfParser = new GLTFParser();
-      const gltf = gltfParser.parse(data);
+      const gltf = gltfParser.parse(data, {createImages: false});
```

Comment:
We should change the default of that option to `false`. Add a TODO to change this when that is done.